src/pages/certificates/create-certificate/create-certificate.js

```diff
@@ -7,37 +10,71 @@ import {
   TextField,
   Tooltip
 } from '@material-ui/core';
-import { useStyles } from './create-certificate.styles';
-import { BackButton } from '../../../components/buttons';
-import materialUiConstants from '../../../configs/material-ui-constants';
-import { useCommonStyles } from '../../common.styles';
-import CheckBoxes from '../checkBoxes';
+
+import {
+  setSnackBarMessage,
+  setSnackBarSeverity,
+  setSnackBarStatus
+} from '../../../redux/snackbar/snackbar.actions';
+
+import LoadingBar from '../../../components/loading-bar';
+
 import formRegExp from '../../../configs/form-regexp';
-import CertificatesTable from '../certificatesTable';
 import { loginErrorMessages } from '../../../configs/error-messages';
 import buttonTitles from '../../../configs/button-titles';
 import titles from '../../../configs/titles';
 import routes from '../../../configs/routes';
+import materialUiConstants from '../../../configs/material-ui-constants';
 
-const { certificatesTitles } = titles;
+import CheckBoxes from '../checkBoxes';
+import { BackButton } from '../../../components/buttons';
+import { useCommonStyles } from '../../common.styles';
+
+import { useStyles } from './create-certificate.styles';
+import CertificatesTable from '../certificatesTable';
+import { bulkGenerateCertificates } from '../operations/certificate.mutation';
+
+const { certificatesTitles, certificatesValueTitles } = titles;
 
 const CreateCertificate = () => {
+  const dispatch = useDispatch();
+
   const commonStyles = useCommonStyles();
   const styles = useStyles();
 
-  const initialCheckboxes = [
-    { checked: false, quantity: 1, name: certificatesTitles[500] },
-    { checked: false, quantity: 1, name: certificatesTitles[1000] },
-    { checked: false, quantity: 1, name: certificatesTitles[1500] }
-  ];
+  const initialCheckboxes = certificatesValueTitles.map((item) => ({
+    checked: false,
+    quantity: 1,
+    ...item
+  }));
 
   const [checkBoxes, setCheckBoxes] = useState(initialCheckboxes);
-  const [date, setDate] = useState(new Date());
+  const [date, setDate] = useState();
   const [email, setEmail] = useState('');
   const [isInvalid, setIsInvalid] = useState(false);
   const [certificates, setCertificates] = useState([]);
   const [disabled, setDisabled] = useState(true);
 
+  const [generateCertificates, { loading: certificatesLoading }] = useMutation(
+    bulkGenerateCertificates,
+    {
+      onCompleted(data) {
+        dispatch(setSnackBarSeverity('success'));
+        dispatch(setSnackBarMessage('Успішно додано'));
+        dispatch(setSnackBarStatus(true));
+
```

Comment:
i think you should use here one dispatch with `showSuccessSnackbar` action instead of three, 
look at my PR  
https://github.com/ita-social-projects/horondi_admin/pull/1213

Reply:
done